### PR TITLE
FedCM updates for Chrome 105

### PR DESCRIPTION
**Pull request by agektmr**
Reflect changes made to FedCM in Chrome 105.

cc: @yi-gu

### CODE REVIEW

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:21]: IIRC we have never communicated publicly that we'd ship FedCM in 105, right? So it's not "postpone" :) More importantly, we have not decided when to ship it. Will probably get an answer in 2-3 weeks.

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:43]: The file name can be `anything.json`

Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:43]: s/url/configUrl

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:240]: Perhaps mention how to get the token from the credential? e.g.
```
const token = credential.token;
```

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:225]: s/ID token issuance/token issuance

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:250]: s/url/configUrl

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:252]: s/with/without

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:318]: greater than or equal to

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:381]: s/url/configUrl

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:397]: why renaming the metadata?

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:600]: s/url/configUrl/

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:641]: "isn't provided" -> "isn't provided OR does not include the RP's client_id AND the user hasn't..."

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:691]: The token can be retrieved from the credential object returned from `n.c.get()`.

File: site/en/blog/fedcm-origin-trial/index.md
Comment (yi-gu) [inline site/en/blog/fedcm-origin-trial/index.md:33]: s/url/configUrl

File: site/en/blog/fedcm-origin-trial/index.md
Comment (agektmr) [reply site/en/blog/fedcm-origin-trial/index.md:691]: This is a removed section.

File: site/en/blog/fedcm-origin-trial/index.md
Comment (agektmr) [reply site/en/blog/fedcm-origin-trial/index.md]: Indeed. Removed this paragraph.

**Comment by yi-gu** [comment]
Thanks! LGTM % one nit

**Comment by agektmr** [comment]
@mihajlija this pull request is ready for your reviews and edits. I'm hoping we can publish this before Chrome 105 goes beta.
https://chromiumdash.appspot.com/schedule

File: site/en/blog/fedcm-origin-trial/index.md
Comment (mihajlija) [inline site/en/blog/fedcm-origin-trial/index.md:31]:
```suggestion
* `url` is now `configURL` and must be a full URL of the manifest JSON file
```

File: site/en/blog/fedcm-origin-trial/index.md
Comment (mihajlija) [inline site/en/blog/fedcm-origin-trial/index.md:21]:
```suggestion
Starting from Chrome 105:
```

File: site/en/blog/fedcm-origin-trial/index.md
Comment (mihajlija) [inline site/en/blog/fedcm-origin-trial/index.md:575]: This doesn't sound very good, "a token is a security token" sounds almost like "a token is a token". 

Could we just edit to "A token is a string that contains claims about the authentication"?

Comment (agektmr) [reply site/en/blog/fedcm-origin-trial/index.md:575]: Yeah, that sounds better.

**Review by mihajlija — APPROVED** [review]

File: site/en/blog/fedcm-origin-trial/index.md
Comment (mihajlija) [inline site/en/blog/fedcm-origin-trial/index.md:575]:
```suggestion
    <td>A token is a string that contains claims about the authentication.</td>
```

**Comment by mihajlija** [comment]
LGTM, proceed to merge when ready.

**Comment by sofiayem** [comment]
@agektmr Just double-checking. Is this ready?

**Comment by agektmr** [comment]
@sofiayem yes, it's ready. Just waiting for Chrome 105 beta to be available.

**Comment by sofiayem** [comment]
I added the "DO NOT MERGE" tag. Remove it and merge when you're ready to publish.

**Pull request merged**
